src/framework/mlt-framework.pc.in

```diff
@@ -4,7 +4,7 @@ libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 datadir=@CMAKE_INSTALL_FULL_DATADIR@
 
-moduledir=${libdir}/mlt-@MLT_VERSION_MAJOR@
```

Comment:
This change is incorrect when MLT is installed on Linux and BSD. See `MLT_INSTALL_MODULE_DIR` in the top level `CMakeLists.txt`.